submissions/paulElugbaju/database/init.sql

```diff
@@ -0,0 +1,55 @@
+-- create new databse called foobbag
+CREATE DATABASE foodbag; 
+
+-- allow foodbag database to be used
+USE foodbag;
+
+-- create table, food
+CREATE TABLE food (
+	id INT NOT NULL AUTO_INCREMENT,
+    type ENUM ('Fast Food','Vegetables & Fruits','Drinks & Cocktails','Resturants'),
```

Comment:
Change line 10 to type ENUM('fast_food', 'vegetable_fruits', 'drinks_cocktails', 'restaurants') NOT NULL DEFAULT 'fast_food. 

When naming tables and columns in SQL, it is recommended to use all lowercase letters and separate words with an underscore. Names should begin with a letter and may not end with an underscore. Only letters, numbers, and underscores should be used in names.
Object names should be clear, meaningful, and descriptive, and a consistent naming convention should be established and followed throughout the database.

https://brainstation.io/learn/sql/naming-conventions
https://www.sqlstyle.guide/
https://www.sqlops.com/best-practices-for-object-naming-convention-in-sql-server/